vitest.config.ts

```
@@ -0,0 +1,13 @@
+import { configDefaults, defineConfig } from 'vitest/config'
+
+export default defineConfig({
+	test: {
+		exclude: [...configDefaults.exclude, '**/*.config.ts'],
+		coverage: {
+			// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+			exclude: [...configDefaults.coverage.exclude!, '**/*.config.ts']
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)